distribution/docs/src/main/resources/_contents/_sources/solrcloud-contents.adoc

```diff
@@ -0,0 +1,109 @@
+==== Solr Cloud
+
+Solr Cloud is a cluster of Solr Server instances that are fault tolerant and highly available.
+Each Solr Server instance in Solr Cloud provides a Solr Admin UI.
+
+Synchronization between Solr Server instances is managed by Zookeeper.
+Zookeeper routes client requests to the proper Solr Server instance in the the Solr Cloud cluster.
+
+.Solr Cloud Deployment
+[ditaa,solrcloud-diagram,png]
+....
+  Container(JVM)         Solr Cloud
+ +=-----------------+   +=--------------------+
+ |                  |   |                     |
+ | /-----------\    |   | +---------+         |
+ | |cDEF       |    |   | |Solr cFF0|         |
+ | | DDF       |    |   | +---------+         |
+ | |           |    |   |                     |
+ | |           |  http  |         +---------+ |
+ | |           +------->|         |Solr cFF0| |
+ | |           |    |   |         +---------+ |
+ | |           |    |   |                     |
+ | |           |    |   | +---------+         |
+ | |           |    |   | |Solr cFF0|         |
+ | \-----------/    |   | +---------+         |
+ |                  |   |                     |
+ +------------------+   +---------------------+
+....
+
+[NOTE]
+====
+Solr Cloud is currently in Beta version.
+It is implemented but has not been tested thoroughly.
```

Comment:
Do we need this line?  I think the beta notice on the previous line is sufficient.